react-components/table-intro/step4.md

```diff
@@ -38,15 +38,15 @@ const defaultRows = [
 
 For each array item in each row's `cells` array, replace the string value with an object whose `title` property is the row cell value.
 
-It should look like this:
+Each array item in each row’s cells array looks like this:
```

Comment:
might help with the awkwardness of the sentence

```suggestion
For every array item in each row’s cells array looks like this:
```

Reply:
Revised this section a bit for clarity, let me know what you think